examples/kubernetes/1.15/cilium.yaml

```diff
@@ -324,13 +311,13 @@ spec:
       - command:
         - /init-container.sh
         env:
-        - name: CILIUM_ALL_STATE
+        - name: CLEAN_CILIUM_STATE
           valueFrom:
             configMapKeyRef:
               key: clean-cilium-state
               name: cilium-config
               optional: true
-        - name: CILIUM_BPF_STATE
+        - name: CLEAN_CILIUM_BPF_STATE
```

Comment:
Note to reviewers, would be good to double-check this as I'm not sure what kind of validation we have for determining that these work in the intended manner.